controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_types.go

```diff
@@ -112,6 +112,10 @@ type KubeadmControlPlaneStatus struct {
 	// state, and will be set to a descriptive error message.
 	// +optional
 	FailureMessage *string `json:"failureMessage,omitempty"`
+
+	// ObservedGeneration is the latest generation observed by the controller.
+	// +optional
```

Comment:
Should this be no longer optional, since we are not setting omitempty? https://github.com/eBay/Kubernetes/blob/master/docs/devel/api-conventions.md#optional-vs-required